types/emoji-mart/dist-es/components/category.d.ts

```diff
@@ -1,12 +1,11 @@
-import React = require('react');
+import * as React from 'react';
 
-import { EmojiData } from '..';
-
-import { Emoji, EmojiProps, I18n } from '.';
+import { Emoji, EmojiData, EmojiProps, I18n } from '..';
 
 export interface Props {
     emojis?: Array<string|EmojiData>;
     hasStickyPosition?: boolean;
+    id: string;
```

Comment:
This is [not in the `propTypes`](https://github.com/missive/emoji-mart/blob/v2.8.0/src/components/category.js#L213-L223) but that [looks like a bug](https://github.com/missive/emoji-mart/blob/v2.8.0/src/components/category.js#L147-L155) on the emoji-mart side.

I also questioned whether this was required or not but [looks like it is](https://github.com/missive/emoji-mart/blob/v2.8.0/src/components/category.js#L189)...

Reply:
Looks like it's supposed to be `keyof I18n['categories']`, or `import { CategoryName } from '../utils/shared-props'`. Probably better to add `CategoryName` to the reexports done in the entry point.